packages/core/admin/admin/src/pages/Home/components/ContentManagerWidgets.tsx

```diff
@@ -73,8 +73,10 @@ const WidgetContent = ({ document }: { document: RecentDocument }) => {
                 id: 'content-manager.widget.last-edited.single-type',
                 defaultMessage: 'Single-Type',
               })
-            : // TODO check how to localize display name
-              document.contentTypeDisplayName}
+            : formatMessage({
+                id: document.contentTypeDisplayName,
```

Comment:
wouldn't the ID be the UID instead of display name?

Reply:
That's what I was hoping but in the Left Menu it's using the displayName. Which I'm not crazy about but not sure we can/should change, either way probably not on this PR.